mandatory/1-fix-functions.js

```diff
@@ -21,7 +21,7 @@ function mood() {
 }
 
 function greaterThan10(num) {
-  let isBigEnough;
+  let isBigEnough = 10;
 
   if (isBigEnough) {
```

Comment:
Also, you need to write `isBigEnough > 10` in your if statement so that it will pass if the number is greater than 10